### DISCUSSION TITLE
Make "Save as copy" generate M2M relations and translations based on the original collection entry

### DISCUSSION THREAD

**Discussion by lluishi93** [directus/directus#2748] [Feature Requests]
At the moment, the "Save as copy" option does not "maintain" the relations and translations from the original entry.

Could it be possible to make a copy (generating new IDS of course) for those as well? For instance, for the M2M, executing new inserts into the junction collections after generating the item and obtaining its ID from the copy.

Same procedure for the translations, using the previous rows in the translations to create new rows, assigning the entry column to the new generated item as a result from the copy.

**Comment by benhaynes** [comment]
Yeah! This is a feature request. Once v9 is out and stable, we'll finally be diving in to the backlog of feature requests.

Duplicate of: https://github.com/directus/app/issues/1276

**Comment by stouch** [comment]
👀

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
